### ISSUE TITLE
Update branch health algorithm to include whether or not a branch has been merged into master

### ISSUE THREAD

**Issue by jwir3** [jwir3/gitbranchhealth#21]
We only support determination of whether a branch is "stale" based on how many days have happened since the last edit. However, it might be useful for some users to indicate a branch is stale if all of the commits in the branch have been merged into the main development branch. We should provide some mechanism for determining this, and utilizing the information in our calculation of healthiness.

This depends on #20.
